woocommerce/woocommerce-admin
# Add card to payments task for WooCommerce Payments

Fixes #3948 

Adds a card for WooCommerce Payments to the Payments Task.

Loose ends
- [ ] Improve detection of configured state (not just relying on plugin activation)
- [ ] Does the "Recommended" black banner get changed in this PR or a separate one?

Related (but not required for merge)

- [ ] https://github.com/Automattic/woocommerce-payments/issues/516 (Allow redirect back to payments task)

Required before release

- [ ] woocommerce-payments in wordpress.org/plugins

### Accessibility




### Screenshots

<img width="1015" alt="active" src="https://user-images.githubusercontent.com/1595739/77597849-56a8d180-6ebd-11ea-88fb-66d035f5c868.png">

<img width="1018" alt="not_active" src="https://user-images.githubusercontent.com/1595739/77597853-59a3c200-6ebd-11ea-934a-6de23104e8c8.png">

### Detailed test instructions:

- Navigate to wp-admin > WooCommerce > Dashboard
- Click on the Payments task
- Observe the prompt to Set Up WooCommerce Payments
- Note: You cannot activate it from this prompt at this time since we haven't published it yet
- Install WooCommerce Payments manually from woocommerce.com or github but do not activate it
- Click on Set up to set up WooCommerce Payments from the task list
- Observe that that completes and you are given the prompt to verify business details
- No need to complete that - just return to wp-admin
- Note: Until 516 is merged to woocommerce payments, you will not be returned to the payments tasks
- Observe that the card now shows a Settings link and the expected toggle button
- Lastly, falsify the wcpay feature flag and ensure this card no longer appears at all

### Changelog Note:

Enhancement: Adds WooCommerce Payments to the Payments Task
